content/enterprise/requirements/data-storage/vault.mdx

```diff
@@ -40,68 +40,71 @@ Terraform Enterprise.
 
   ```hcl
   # To renew leases.
-  path "sys/leases/renew/*" {
-    capabilities = ["create", "read", "update"]
+  path "sys/leases/renew" {
+    capabilities = ["create", "update"]
   }
-  path "sys/renew/*" {
-    capabilities = ["create", "read", "update"]
+  path "sys/renew" {
+    capabilities = ["create", "update"]
   }
 
   # To renew tokens.
-  path "auth/token/renew/*" {
-    capabilities = ["create", "read", "update"]
+  path "auth/token/renew" {
+    capabilities = ["create", "update"]
+  }
+  path "auth/token/renew-self" {
+    capabilities = ["create", "update"]
   }
```

Comment:
Added this path since it's used to renew the periodic token in Terraform Enterprise.